Framework/API/src/Sample.cpp

```diff
@@ -152,31 +152,76 @@ void Sample::setEnvironment(std::shared_ptr<SampleEnvironment> env) { m_environm
  * @throw std::runtime_error If the OrientedLattice has not been defined
  */
 const OrientedLattice &Sample::getOrientedLattice() const {
-  if (!m_lattice) {
+  if (!m_lattices) {
     throw std::runtime_error("Sample::getOrientedLattice - No OrientedLattice has been defined.");
   }
-  return *m_lattice;
+  return (*m_lattices)[0];
```

Comment:
Ah yes I see it returns the first element!

Reply:
1. yes, they will return the same object, `getOrientedLattice` will always return the first element. Also, note that `setOrientedLattice` will reset the list and add one lattice to it, this behaviour is defined to ensure backward compatibility. You can set multiple oriented lattices using `setOrientedLattices`.
2. yes, it will always return the first element